packages/wrangler/src/experimental-assets.ts

```diff
@@ -412,6 +412,15 @@ export function verifyMutuallyExclusiveAssetsArgsOrConfig(
 			"Cannot use Experimental Assets and tail consumers in the same Worker. Tail Workers are not yet supported for Workers with assets."
 		);
 	}
+
+	if (
+		(args.experimentalAssets || config.experimental_assets) &&
+		(args as StartDevOptions).remote
```

Comment:
Rather than casting here we should ensure that this value gets passed in by changing the signature of `verifyMutuallyExclusiveAssetsArgsOrConfig()`.

Reply:
I'll move this check out of this fn instead. This is something we only need to check in dev, whereas `verifyMutuallyExclusiveAssetsArgsOrConfig` is called in multiple places

Reply:
Nice!